develop/parachain-devs/runtime-development/runtime-apis.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Runtime APIs
+description: Highlights the runtime interfaces that enable communication with outer node services.  These APIs allow for the outer node to communicate with the WebAssembly runtime of a Polkadot SDK-based node.
+---
+
+Nodes consist of outer node services and a runtime. This separation of responsibilities is an important concept for designing Polkadot SDK-based chains and building upgradeable logic.
+
+For more information on details of how the meta protocol works, [refer to the Polkadot SDK docs.](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html#summary){target=_blank}
+
+However, the outer node services and the runtime must communicate with each other to complete many critical operations, including reading and writing data and performing state transitions.
+
+The outer node services communicate with the runtime by calling runtime application programming interfaces to perform specific tasks.
+
+By default, the runtime provides the following traits for outer node services to call. Below is a non-exhaustive list of common runtime APIs:
+
+- [`AccountNonceApi`](https://paritytech.github.io/polkadot-sdk/master/frame_system_rpc_runtime_api/trait.AccountNonceApi.html){target=_blank}
```

Comment:
I'd add a small description for each of these items that clarifies what they do. I found something about this here: https://docs.substrate.io/reference/runtime-apis/